python_modules/libraries/dagster-dbt/dagster_dbt/components/dbt_project/component.py

```diff
@@ -128,12 +128,13 @@ class DbtProjectComponent(Component, Resolvable):
     translation: Optional[ResolvedTranslationFn] = None
     select: str = "fqn:*"
     exclude: Optional[str] = None
+    settings: Optional[DagsterDbtTranslatorSettings] = None
```

Comment:
`translator_settings` maybe, felt weird next to `translation` when i was writing this